src/commonMain/kotlin/fr/acinq/lightning/wire/LightningMessages.kt

```diff
@@ -1747,6 +1751,50 @@ data class PhoenixAndroidLegacyInfo(
     }
 }
 
+data class DNSAddressRequest(val offer: OfferTypes.Offer, val languageSubtype: String) : LightningMessage {
+
+    override val type: Long get() = DNSAddressRequest.type
+
+    override fun write(out: Output) {
+        val serializedOffer = offer.encode()
+        LightningCodecs.writeU16(serializedOffer.length, out)
+        LightningCodecs.writeBytes(serializedOffer.toByteArray(charset = Charsets.UTF_8), out)
+        LightningCodecs.writeU16(languageSubtype.length, out)
+        LightningCodecs.writeBytes(languageSubtype.toByteArray(charset = Charsets.UTF_8), out)
+    }
+
+    companion object : LightningMessageReader<DNSAddressRequest> {
+        const val type: Long = 35025
```

Comment:
This value is also used by on-the-fly funding for the `recommended_feerates` message (in https://github.com/ACINQ/lightning-kmp/pull/660). I should change that, and use a value that's further from the current extension messages we used that aren't specified.